M2/Macaulay2/packages/MonomialIntegerPrograms.m2

```diff
@@ -0,0 +1,1656 @@
+-- -*- coding: utf-8 -*-
+-- Copyright (C) 2020 Lily Silverstein and Jay White
+--
+-- This program is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation, either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+newPackage (
+  "MonomialIntegerPrograms",
+  Version=>"1.0",
+  Authors => {
+      {Name => "Lily Silverstein", Email => "lsilverstein@cpp.edu", HomePage => "https://cpp.edu/faculty/lsilverstein"},
+      {Name => "Jay White", Email => "jay.white@uky.edu", HomePage => "https://math.as.uky.edu/users/jwh245"}
+      },
+  Headline => "using integer programming for fast computations with monomial ideals",
+  Configuration => {
+      "CustomPath" => "",
+      "CustomScipPrintLevel" => ""
+      },
+  PackageImports => {"LexIdeals"},
+  AuxiliaryFiles => true,
+  CacheExampleOutput => true,
+  DebuggingMode => true
```

Comment:
DebuggingMode should be false.

Reply:
fixed with commit 169fdc25e4e731c7f8c59fb610e18160654e008d